src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/PrisonApiService.kt

```diff
@@ -72,6 +72,11 @@ class PrisonApiService(
     return levelsMap.toMap()
   }
 
+  fun hasPrisonerGotEnhancedPrivilege(prisonerId: String): Boolean {
+    // TODO This must implemented by another technical task this is just a dummy call
+    return true
+  }
+
```

Comment:
This is for you @dhirajshettymoj to fill in you might want to move it to another service or call it PrisonerService I guess because it just gets details about the prisoner

Reply:
yes makes sense. Will do that in my branch.